styles.css

```diff
@@ -0,0 +1,75 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-link {
+  margin: 2%;
+  color: #6070ff;
+  margin-top: 10%;
+}
```

Comment:
- [ ] Please ensure you have matched the design for the header section.

Implementation             |  Figma Design
:-------------------------:|:-------------------------:
![nav_implementation](https://user-images.githubusercontent.com/45002654/125027383-d5b02200-e08e-11eb-8089-9fc25fd7aedd.PNG)  |  ![nav_design](https://user-images.githubusercontent.com/45002654/125027408-dea0f380-e08e-11eb-9522-03854f5c07b3.PNG)